bessctl/commands.py

```diff
@@ -762,9 +762,9 @@ def run_file(cli, conf_file, env_map):
     _run_file(cli, os.path.expanduser(conf_file), env_map)
 
 
-@cmd('add worker WORKER_ID CORE', 'Create a worker')
-def add_worker(cli, wid, core):
-    cli.bess.add_worker(wid, core)
+@cmd('add worker WORKER_ID CORE [SCHEDULER]', 'Create a worker')
```

Comment:
I think the "[SCHEDULER]" keyword needs to be defined, otherwise the command interpreter expects a literal "[SCHEDULER].  I had to add this:

```
diff --git a/bessctl/commands.py b/bessctl/commands.py
index 382b7a7..59701d2 100644
--- a/bessctl/commands.py
+++ b/bessctl/commands.py
@@ -253,6 +253,11 @@ def get_var_attrs(cli, var_token, partial_word):
             var_type = 'name'
             var_desc = 'specify a name of the new port'
 
+        elif var_token == '[SCHEDULER]':
+            var_type = 'name'
+            var_desc = 'specify the type of scheduler'
+            var_candidates = ['fast']
+
         elif var_token == 'PORT':
             var_type = 'name'
             var_desc = 'name of a port'
```